### PR TITLE
change native token to HKO/PARA

### PR DESCRIPTION
**Pull request by GopherJ**
frontend is implmenting tx fee, so we need to add new tokens: PARA/HKO

companion pr: https://github.com/parallel-finance/parallel-dapp/pull/307

### CODE REVIEW

**Comment by GopherJ** [comment]
we shouldn't merge this util it's tested and the `PARA/HKO` token's icon has been designed

File: runtime/vanilla/src/lib.rs
Comment (kaichaosun) [inline runtime/vanilla/src/lib.rs:522]: This should be something like `DEV`.

**Review by kaichaosun — APPROVED** [review]

**Comment by crystalynliu** [comment]
After the PR merged, we should also change the decimal and token name with configuration.

https://github.com/parallel-finance/parallel-dapp/blob/testnet/src/transaction/index.js#L59
https://github.com/parallel-finance/parallel-dapp/blob/testnet/src/pages/LendAndBorrow/Transaction/index.js#L156

File: runtime/vanilla/src/lib.rs
Comment (GopherJ) [reply runtime/vanilla/src/lib.rs:522]: @yz89 what do we want exactly?

Comment (GopherJ) [reply runtime/vanilla/src/lib.rs:522]: @yz89 do we really need a currency for vanilla?

Comment (GopherJ) [reply runtime/vanilla/src/lib.rs:522]: looks like acala doesn't have a currency for mandala https://github.com/AcalaNetwork/Acala/blob/master/primitives/src/currency.rs#L175-L193 @yz89 @kaichaosun

Comment (yz89) [reply runtime/vanilla/src/lib.rs:522]: Ok, I now think it make sense to use `HKO`. It shouldn't to take a place on CurrencyId.

**Pull request merged**
